ProjectBank/Core/IProjectRepository.cs

```diff
@@ -1,5 +1,10 @@
 namespace ProjectBank.Core;
-
-public class IProjectRepository
+public interface IProjectRepository
 {
+    Task<ProjectDTO> CreateProjectAsync(ProjectCreateDTO project);
+    Task<IReadOnlyCollection<ProjectDTO>> ReadAllAsync();
+    Task<ProjectDTO> ReadProjectAsync(int projectId);
+    Task<IReadOnlyCollection<ProjectDTO>> ReadProjectsByUserAsync(int userId);
+    Task<Response> UpdateProjectAsync(ProjectDTO project);
```

Comment:
Skal denne måske deles op i to? 
Én til når en supervisor "EditProjectAsync(ProjectDTO project)", og Én til når project ændre status fra Active til Closed
"UpdateStatusProjectAsync(int projectId)"

Reply:
Jeg tror også vi skal have en der returnerer alle projects med bestemt tag